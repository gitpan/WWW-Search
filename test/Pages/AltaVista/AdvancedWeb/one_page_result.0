<html><head>
<title>AltaVista: Advanced Query LSAM AND AutoSearch</title>
</head><body>
<font face="Arial,Helvetica" size="+1"><a href="/cgi-bin/query?text&pg=q&what=web">AltaVista Search Service</a></font><p><FORM name=mfrmaction="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=aq><INPUT TYPE=hidden NAME=text VALUE=yes><font size=-1><B>Enter ranking keywords in</B>
<SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><font size=-1><p><input type=text name=r size=50 value="LSAM AutoSearch">
<INPUT TYPE=submit name=search VALUE="Search"><br>
 Enter boolean expression:<br>
<textarea name=q cols=50 rows=3 maxlength=800 wrap=virtual>LSAM AND AutoSearch</textarea><p>Range of Dates (e.g. 21/Mar/96): <b> From </b><INPUT TYPE=text NAME=d0 SIZE=9 MAXLENGTH=32 VALUE="">
<b> To </b><INPUT TYPE=text NAME=d1 SIZE=9 MAXLENGTH=32 VALUE=""><p><INPUT TYPE=checkbox name=fmt value=n> <b>Count documents matching the boolean expression.</b> 
<P><font size=-1><a href="/cgi-bin/query?pg=q&what=web&text=yes">Search</a> - Advanced Text Search - <a href="/cgi-bin/query?pg=aq&what=news&text=yes">Usenet</a></font></font></FORM>
AltaVista found 7 Web pages. <P><font size=-1><br>
</font><font size=-1><dl><dt><b>1. </b><a href="http://www.isi.edu/div7/lsam/tools/autosearch/index.html"><b>index.html directory page</b></a><dd>
AutoSearch WEB Searching. What is AutoSearch? AutoSearch performs a web-based search and puts the results set in a web page. It periodically updates this..<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/tools/autosearch/index.html<br>
Last modified 3-Jun-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2ftools%2fautosearch%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>2. </b><a href="http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/bin/AutoSearch.html"><b>AutoSearch -- a web-searching application</b></a><dd>
Contained in JOHNH/WWW-Search-1.023.tar.gz. requires perl 5.004 or later. NAME. SYNOPSIS. DESCRIPTION. OPTIONS. DESCRIPTION. AUTOMATED SEARCHING. CHANGING.<br><b>URL:</b> <font color=gray>theory.uwinnipeg.ca/CPAN/data/WWW-Search/bin/AutoSearch.html<br>
Last modified 2-Apr-99 - page size 18K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2ftheory%2euwinnipeg%2eca%2fCPAN%2fdata%2fWWW%2dSearch%2fbin%2fAutoSearch%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>3. </b><a href="http://www.isi.edu/lsam/tools/autosearch/TEST/index.html"><b>Test Pages for AutoSearch</b></a><dd>
Test Pages for AutoSearch. Singleton. Dozen: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12. hidden anchor page. AutoSearch-page. Page maintainer: John Heidemann...<br><b>URL:</b> <font color=gray>www.isi.edu/lsam/tools/autosearch/TEST/index.html<br>
Last modified 27-Nov-96 - page size 1K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2flsam%2ftools%2fautosearch%2fTEST%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>4. </b><a href="http://www.isi.edu/people/johnh/WORK/index.html"><b>John Heidemann's Work</b></a><dd>
John Heidemann's Work. I am computer scientist at Information Sciences Institute and a research assistant professor at the University of Southern...<br><b>URL:</b> <font color=gray>www.isi.edu/people/johnh/WORK/index.html<br>
Last modified 20-Nov-98 - page size 7K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fpeople%2fjohnh%2fWORK%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>5. </b><a href="http://gost.isi.edu/people/johnh/WORK/index.html"><b>John Heidemann's Work</b></a><dd>
John Heidemann's Work. I am computer scientist at Information Sciences Institute of the University of Southern California. I will also be teaching at USC..<br><b>URL:</b> <font color=gray>gost.isi.edu/people/johnh/WORK/index.html<br>
Last modified 25-Jun-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fgost%2eisi%2eedu%2fpeople%2fjohnh%2fWORK%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>6. </b><a href="http://www.isi.edu/~wls/"><b>William L. Scheding @ ISI</b></a><dd>
William L. (Bill) Scheding (Resume) Systems Programmer. USC/Information Sciences Institute. 4676 Admiralty Way, Suite 1001. Marina del Rey, CA 90292-6695..<br><b>URL:</b> <font color=gray>www.isi.edu/~wls/<br>
Last modified 15-Mar-99 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7ewls%2f&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>7. </b><a href="http://www.isi.edu/div7/lsam/tools/WWW_SEARCH/index.html"><b>WWW::Search</b></a><dd>
WWW::Search. WWW::Search is a collection of Perl modules which provide an API to WWW search engines like AltaVista, Lycos, Hotbot, WebCrawler, and so on...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/tools/WWW_SEARCH/index.html<br>
Last modified 11-Dec-98 - page size 12K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2ftools%2fWWW%5fSEARCH%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
</font><font size=-1><br>
</font><font size=-2>word count: LSAM: 817; AutoSearch: 1359<BR></font><P>
<p><font face="Arial,Helvetica" size="-2">
<a href="/">AltaVista.com</a> |
<a href="/av/content/about.htm">About AltaVista</a> |
<a href="/av/content/help.htm">Help</a> |
<a href="/av/content/questions.htm">Feedback</a> |
<a href="http://www.doubleclick.net/advertisers/altavista/">Advertising Info</a> |
<a href="/cgi-bin/query?pg=addurl">Add a Page</a><br>
<a href="/av/content/disclaimer.htm">Disclaimer</a> |
<a href="/av/content/privacy.htm">Privacy</a> |
<a href="http://image.altavista.com/AV_CopyrightPolicy.htm">Copyright</a> |
<a href="/av/content/av_network.html">International</a> |
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> |
<a href="/cgi-bin/query">Graphic Version</a></font>
</BODY></HTML>
