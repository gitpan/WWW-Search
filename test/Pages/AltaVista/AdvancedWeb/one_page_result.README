This set of test-result pages was created on Mon Aug 16 16:18:34 1999
