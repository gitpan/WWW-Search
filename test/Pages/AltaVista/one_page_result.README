This set of test-result pages was created on Wed Sep 29 16:32:56 1999
