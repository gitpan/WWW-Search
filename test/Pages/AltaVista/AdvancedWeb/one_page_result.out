http://www.isi.edu/div7/cities/tools/autosearch/index.html
http://www.isi.edu/~lsam/tools/autosearch/TEST/index.html
http://gost.isi.edu/people/johnh/WORK/index.html
http://www.engelschall.com/ar/perldoc/pages/util/AutoSearch.html
http://www.isi.edu/div7/cities/tools/WWW_SEARCH/index.html
http://gin.isi.edu/lsam/tools/autosearch/TEST/index.html
http://gin.isi.edu/~johnh/WORK/index.html
http://sofa.dartnet.co.uk/manual/perl5/site_perl/WWW/autosearch.pl.html
http://www.isi.edu/~wls/index.html
